imports/ui/components/root-app-bar.jsx

```diff
@@ -13,7 +13,7 @@ import {
 } from '../components/app-bar.mui-styles'
 
 const RootAppBar = ({ title, onIconClick }) => (
-  <AppBar title={title} titleStyle={titleStyle}
+  <AppBar title={title} titleStyle={titleStyle} style={{boxShadow: 'none'}}
```

Comment:
Don't use inline styles. Also, this component is obviously used in many other places, you shouldn't make changes that remove something that's undesirable for one usage, but is desirable for the others